Dockerfile

```diff
@@ -30,6 +30,7 @@ RUN apt-get -q -y update \
     libproj-dev \
     libgdal-dev \
     libsqlite3-dev \
+    xapp \
```

Comment:
Can you elaborate a bit more what this solves?

Reply:
I'm trying to follow: `doc/tutorials/Create_WMS_on_series_KNMI_HDF5_radar_files.md`. When in step 4 on master branch, 

```
(base) eveline@eevee-dell ~/code/adaguc-server/Docker (master)$ ls ${ADAGUC_DATA_DIR}/
HARM_N25_20171215090000_dimx16_dimy16_dimtime49_dimforecastreferencetime1_varairtemperatureat2m.nc  RAD_NL25_PCP_NA_202312181330.h5
RAD_NL25_PCP_NA_202312181310.h5                                                                     RAD_NL25_PCP_NA_202312181335.h5
RAD_NL25_PCP_NA_202312181315.h5                                                                     RAD_NL25_PCP_NA_202312181340.h5
RAD_NL25_PCP_NA_202312181320.h5                                                                     RAD_NL25_PCP_NA_202312181350.h5
RAD_NL25_PCP_NA_202312181325.h5                                                                     RAD_NL25_PCP_NA_202312181355.h5
(base) eveline@eevee-dell ~/code/adaguc-server/Docker (master)$ ls ${ADAGUC_DATASET_DIR}/
edr.xml  RAD_NL25_PCP_CM.xml
base) eveline@eevee-dell ~/code/adaguc-server/Docker (master)$ docker exec -i -t my-adaguc-server /adaguc/adaguc-server-updatedatasets.sh RAD_NL25_PCP_CM
<<OK output>>   ***** Finished DB Update *****

(base) eveline@eevee-dell ~/code/adaguc-server/Docker (master)$ . .env && firefox ${EXTERNALADDRESS}
Gtk-Message: 13:45:20.813: Failed to load module "xapp-gtk3-module"
Gtk-Message: 13:45:20.813: Not loading module "atk-bridge": The functionality is provided by GTK natively. Please try to not load it.
Gtk-Message: 13:45:20.813: Failed to load module "appmenu-gtk-module"
```
and then it also doesn't seem to load correctly? This is on my new KNMI laptop with Ubuntu LTS.

Installing the xapp package seems to solve the error with the gtk3 module.

Reply:
Thanks, that is interesting. I do not completely understand why yet. :eye:

Reply:
The last three logs line (`Gtk-Message`), do they come from Firefox or from the Docker container.

It seems strange that installing `xapp` package (I assume it is [this](https://github.com/linuxmint/xapp)) **in** the docker container solves a desktop related error.

But I must be missing something, as I can assume you had a good reason to add it to the docker container (and not just install it on your machine).

Reply:
I tried the radar tutorial on my machine (Mac) using the released adaguc image and had no problems.

Reply:
With or without xapp I have similar errors, but I think they come from firefox.

My suggestion is to leave the dockerfile as is, but update the documentation with the suggestions from Eveline (:pray:) :smile:

Reply:
Thanks!